src/__tests__/utils/ChatMessageUtils.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import * as ChatMessageUtils from '../../utils/ChatMessageUtils';
+
+it('check getSUSIMessageData util', () => {
+    var timestamp = Date.now();
+    var message = {
+        id: 'm_' + timestamp,
+        threadID: "t_1",
+        authorName: 'SUSI', // hard coded for the example
+        text: 'Hello',
+        response: {},
+        date: new Date(timestamp),
+        isRead: true,
+        responseTime: 123
+    }
+    expect(ChatMessageUtils.getSUSIMessageData(message, "t_1").id).toBe(message.id);
+    expect(ChatMessageUtils.getSUSIMessageData(message, "t_1").threadID).toBe(message.threadID);
+    expect(ChatMessageUtils.getSUSIMessageData(message, "t_1").authorName).toBe(message.authorName);
+    expect(ChatMessageUtils.getSUSIMessageData(message, "t_1").text).toBe(message.text);
+    expect(ChatMessageUtils.getSUSIMessageData(message, "t_1").response).toBe(message.response);
+});
+
+it('check getCreatedMessageData util', () => {
+    var timestamp = Date.now();
+    var messageText = "Hello";
+    var currentThreadID = "t_1";
+    var message = {
+        id: 'm_' + timestamp,
+        threadID: currentThreadID,
```

Comment:
I think this would just increase redundancy, I mean duplicate code for message, what do you think ?  @isuruAb 
If we change structure of message in utils, we need to change this as well.